quickwit/quickwit-proto/protos/quickwit/metastore.proto

```diff
@@ -240,6 +243,33 @@ message IndexMetadataResponse {
   string index_metadata_serialized_json = 1;
 }
 
+message IndexesMetadataRequest {
+  repeated IndexMetadataSubrequest subrequests = 1;
+}
+
+message IndexMetadataSubrequest {
+  optional string index_id = 1;
+  optional quickwit.common.IndexUid index_uid = 2;
+}
+
+message IndexesMetadataResponse {
+  // A JSON serialized then ZSTD compressed list of `IndexMetadata`: `Vec<IndexMetadata> | JSON | ZSTD`.
```

Comment:
We don't use repeated here to make sure we get a better compression rate?
(it could be worth commenting)

Reply:
I am not sure about Tonic but I believe gRPC  supports per method compression. Should we see if we can move zstd compression to the higher layer maybe?

Reply:
- Yes. I'll add a comment.
- Unfortunatley, tonic does not support per method compression. It's all or nothing. We could have a custom codec based on prost codec where compression kicks once the size of a message is larger than some predefined threshold.

Reply:
thanks for the info.